### PR TITLE
Nodejs

### PR DESCRIPTION
**Pull request by abharana24**
implemented JWT token in nodejs. performed CRUD operations.

### CODE REVIEW

File: Abha_Rana/login_signup/html/admin.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```
Comment (vishivish18) [inline Abha_Rana/login_signup/html/admin.html:1]: Are you sure you are aware of the row-column structure of bootstrap? If no please try and go through it again

File: Abha_Rana/login_signup/html/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+			</div>
+			<div class="card-footer">
+				<div class="d-flex justify-content-center links">
+					Don't have an account?<a href="signup.html">Sign Up</a>
+        </div>
+			</div>
+		</div>
+	</div>
+</div>
+<script src="../scripts/sc1.js"></script>
```
Comment (vishivish18) [inline Abha_Rana/login_signup/html/index.html:52]: What does `sc1.js` mean?

File: Abha_Rana/login_signup/scripts/adminsc.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  xhr.send();
+  xhr.onload = function() {
+    if (xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
+      var userdata=JSON.parse(xhr.response);
+      console.log(userdata);
+  var table = document.getElementById("empTable");
+       for(let i=0; i<userdata.length; i++){
+        let tr = document.createElement("tr");
+        
+        var x = userdata[i];
```
Comment (vishivish18) [inline Abha_Rana/login_signup/scripts/adminsc.js:25]: x ??? Variable named X ?

File: Abha_Rana/login_signup/scripts/adminsc.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        tr.appendChild(ad);
+        tr.appendChild(pr);
+        
+        table.appendChild(tr);
+    
+}
+}
+}
+}
+function DecodeToken(token) {
```
Comment (vishivish18) [inline Abha_Rana/login_signup/scripts/adminsc.js:70]: why does some your function name start in lower case some in upper case ?

File: Abha_Rana/login_signup/scripts/adminsc.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  xhr.onload = function() {
+    if (xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
+      var userdata=xhr.response;
+      console.log(userdata);
+      window.alert(userdata);
+}
+  }
+}
+const updateuser = () => {
+  var n=window.prompt("enter name :","Name");
```
Comment (vishivish18) [inline Abha_Rana/login_signup/scripts/adminsc.js:148]: n? e ? a ? ob??

**Comment by vishivish18** [comment]
@abharana24 
Please look at your naming convention, correct the code and raise a PR again
Let me know if you have any doubts

**Comment by abharana24** [comment]
> @abharana24
> Please look at your naming convention, correct the code and raise a PR again
> Let me know if you have any doubts

**Comment by abharana24** [comment]
-have resolved all issues, removed dot net code snippets and raised a PR again with a different branch named nodejwt. kindly review that branch.

**Pull request closed without merging**